src/AutoRest.CSharp/Common/Output/Builders/BuilderHelpers.cs

```diff
@@ -220,6 +221,21 @@ public static CSharpType GetTypeFromExisting(ISymbol existingMember, CSharpType
             return PromoteNullabilityInformation(newType, defaultType);
         }
 
+        public static InputType GetInputTypeFromCSharpType(CSharpType type)
+        {
+            if (TypeFactory.IsList(type))
+            {
+                return new InputListType("Array", GetInputTypeFromCSharpType(type.Arguments[0]), TypeFactory.IsReadOnlyMemory(type), false);
+            }
+
+            if (TypeFactory.IsDictionary(type))
+            {
+                return new InputDictionaryType("Dictionary", InputPrimitiveType.String, GetInputTypeFromCSharpType(type.Arguments[1]), false);
+            }
+
+            return InputPrimitiveType.Object;
+        }
```

Comment:
because of the reason I said [here](https://github.com/Azure/autorest.csharp/pull/4435/files#r1535335683), please move this method back to where it was.

Reply:
well I just noticed that because of a recent update, this method is not used anywhere, therefore we should remove it.

Reply:
removed